docs/getting-started-guides/windows/index.md

```diff
@@ -1,144 +1,262 @@
 ---
-title: Windows Server Containers
+title: Using Windows Server Containers in Kubernetes
 ---
+**Note:** These instructions were recently updated based on Windows Server platform enhancements and the Kubernetes v1.9 release
 
-Kubernetes version 1.5 introduces support for Windows Server Containers. In version 1.5, the Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server.
+Kubernetes version 1.5 introduced Alpha support for Windows Server Containers based on the Windows Server 2016 operating system. With the release of Windows Server version 1709 and using Kubernetes v1.9 users are able to deploy a Kubernetes cluster either on-premises or in a private/public cloud using a number of different network topologies and CNI plugins. Some key feature improvements for Windows Server Containers on Kubernetes include:
+- Improved support for pods! Shared network namespace (compartment) with multiple Windows Server containers (shared kernel)
+- Reduced network complexity by using a single network endpoint per pod
+- Kernel-Based load-balancing using the Virtual Filtering Platform (VFP) Hyper-v Switch Extension (analogous to Linux iptables)
+- Container Runtime Interface (CRI) pod and node level statistics
+- Support for kubeadm commands to add Windows Server nodes to a Kubernetes environment
 
-**Note:** Windows Server Containers on Kubernetes is an Alpha feature in Kubernetes 1.5.
+The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server 2016 or later
+
+**Note:** Windows Server Containers on Kubernetes is a Beta feature in Kubernetes v1.9
+
+## Build
+We recommend using the release binaries that can be found at [https://github.com/kubernetes/kubernetes/releases](https://github.com/kubernetes/kubernetes/releases). Look for the Node Binaries section by visiting the binary downloads link. 
+
+If you wish to build the code yourself, please follow the next instructions:
+
+1. Install the pre-requisites on a Linux host:
+
+```
+sudo apt-get install curl git build-essential docker.io conntrack
+```
+
+2. Run the following commands to build kubelet and kube-proxy:
```

Comment:
Looks like it's resetting the numbering because of the code-block in step 1. Try indenting it all (including the triple-backticks) two spaces? Not sure.

Reply:
addressed